x-pack/test/api_integration/apis/security_solution/saved_objects/draft_timeline.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+
+  describe('Draft timeline - Saved Objects', () => {
```

Comment:
There were no FTR tests for the draft timeline route at all and parts of it were actually broken (see https://github.com/elastic/kibana/pull/178212/commits/633b8b6473bccaa4c9fd5640297943d6d2f9ee08).